tools/lsp/Cargo.toml

```diff
@@ -21,6 +21,7 @@ sixtyfps-compilerlib = { version = "=0.0.6", path = "../../sixtyfps_compiler"}
 sixtyfps-interpreter = { version = "=0.0.6", path = "../../sixtyfps_runtime/interpreter"}
 sixtyfps-corelib = { version = "=0.0.6", path = "../../sixtyfps_runtime/corelib"}
 sixtyfps-rendering-backend-default = { version = "=0.0.6", path="../../sixtyfps_runtime/rendering_backends/default" }
+sixtyfps = { version = "=0.0.6", path = "../../api/sixtyfps-rs"}
```

Comment:
Not sure how I feel about this, this add quite some dependency to the lsp, while we will never get rid of the private dependencies anywy